README.rst

```diff
@@ -81,14 +81,16 @@ Integrade can be configured to test any instance of cloudigrade. The
     CLOUDIGRADE_BASE_URL # base url without http/https prefix
     CLOUDIGRADE_TOKEN    # This is the token of the super user created for
                          # cloudigrade
+    CLOUDIGRADE_CUSTOMER_ACCT # account # for "customer" AWS account
+    CLOUDIGRADE_CUSTOMER_ROLE_ARN # arn for role to grant cloudigrade
+                                  # access to the customer account for
+                                  # metering purposes
+    CLOUDIGRADE_CUSTOMER_ROLE_ARN_2 # additional arn, can be to same account
```

Comment:
I think instead of having many variables as we need more accounts what if a list was passed instead? Something like:

```bash
export CLOUDIGRADE_CUSTOMER_ROLE_ARNS="arn1,arn2,arnN..."

Reply:
I like that, thanks for the suggestion.

I think we are quickly moving towards a time when we may need more complex configuration that could be provided by a file, but until this this is much more flexible

Reply:
Yeah, having a config file will definitely be a thing in the future.